# srv file for global planner.
# is used to request a new global path from a start to a goal.

Header header
float32[] start_state
float32[] goal_pos
float32   goal_tol
---
float32[] waypoint_x
float32[] waypoint_y
